spotinst_sdk2/models/elastigroup/gcp/__init__.py

```diff
@@ -207,142 +218,87 @@ class AutoScale:
     labels: list[Label]
```

Comment:
labels field is not present as per documentation. Please remove.

Reply:
Postman returns labels as a parameter

Reply:
Removed